2996_reflection/reflection.md

```diff
@@ -1504,6 +1504,16 @@ We propose `[:` and `:]` be single tokens rather than combinations of `[`, `]`,
 Among others, it simplifies the handling of expressions like `arr[[:refl():]]`.
 On the flip side, it requires a special rule like the one that was made to handle `<::` to leave the meaning of `arr[::N]` unchanged and another one to avoid breaking a (somewhat useless) attribute specifier of the form `[[using ns:]]`.
 
+A syntax that is delimited on the left and right is useful here because spliced expressions may involve lower-precedence operators.
```

Comment:
Let's also add this sentence which I thought we had somewhere but we don't. 
```suggestion
A syntax that is delimited on the left and right is useful here because spliced expressions may involve lower-precedence operators. Additionally, it's important that the left- and right-hand delimiters are different so as to allow nested splices when that comes up. 
```